core/controllers/profile.py

```diff
@@ -258,7 +255,6 @@ def get(self):
             return
 
         self.values.update({
-            'meta_description': feconf.SIGNUP_PAGE_DESCRIPTION,
             'CAN_SEND_EMAILS': feconf.CAN_SEND_EMAILS,
```

Comment:
Do we need this here? Do we actually use it in the signup page HTML?

Reply:
Done